src/gui/search/searchwidget.cpp

```diff
@@ -141,6 +146,24 @@ SearchWidget::SearchWidget(MainWindow *mainWindow)
     connect(focusSearchHotkey, &QShortcut::activated, this, &SearchWidget::toggleFocusBetweenLineEdits);
 }
 
+// Close tabs when middle-clicked
```

Comment:
This shouldn't be here, move it near to the related code.

Reply:
I was putting it there to explain the purpose of the event filter.

Reply:
>I was putting it there to explain the purpose of the event filter.

Since the event filter is installed, it now become a generic component for the tabBar. Under this viewpoint, this comment will become inappropriate when someone expands its functionality later (if it ever happens).